Test Failure: junit.framework.TestSuite.com.ibm.ws.testing.opentracing.test.FATSuite

RTC 289401

```
com.ibm.ws.testing.opentracing.test.FATSuite:java.lang.RuntimeException: Error getting server
    at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:204)
    at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:97)
    at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:79)
    at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:56)
    at componenttest.rules.repeater.FeatureReplacementAction.setup(FeatureReplacementAction.java:559)
    at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:142)
Caused by: componenttest.exception.TopologyException: Unable to delete pre-existing server directory: C:/Users/jazz_build/Build/jazz-build-engines/wasrtc/EBCPROD/build/dev/image/output/wlp/usr/servers/opentracingFATServer4
    at componenttest.topology.impl.LibertyServer.setup(LibertyServer.java:658)
    at componenttest.topology.impl.LibertyServer.<init>(LibertyServer.java:599)
    at componenttest.topology.impl.LibertyServerFactory.getLibertyServer(LibertyServerFactory.java:160)
```

I think the problem was `.gitignore` was held by a process so it couldn't be deleted on Windows platform.

```
[03/05/2022 03:43:13:195 PST] 001 RemoteFile                     basicDeleteLocalFile           I Failed to delete 'C:\Users\jazz_build\Build\jazz-build-engines\wasrtc\EBCPROD\build\dev\image\output\wlp\usr\servers\opentracingFATServer2\.gitignore': C:\Users\jazz_build\Build\jazz-build-engines\wasrtc\EBCPROD\build\dev\image\output\wlp\usr\servers\opentracingFATServer2\.gitignore: The process cannot access the file because it is being used by another process
```

--- a/dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java
+++ b/dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java
@@ -91,7 +91,7 @@ public class RemoteFile {
     /**
      * The amount of time to wait before retrying
      */
-    public static final long STANDARD_RETRY_INTERVAL_NS = TimeUnit.MILLISECONDS.toNanos(100); // 0.1s
+    public static final long STANDARD_RETRY_INTERVAL_NS = TimeUnit.MILLISECONDS.toNanos(200); // 0.1s
 
     public interface Operation {
         public boolean act() throws Exception;

